web/src/utils/index.ts

```diff
@@ -9,17 +9,12 @@ export function sleep(ms: number) {
 export function baseUrl() {
   let baseUrl = "";
   if (window.APP.baseUrl) {
-    if (window.APP.baseUrl === "/") {
-      baseUrl = "/";
-    } else if (window.APP.baseUrl === "{{.BaseUrl}}") {
-      baseUrl = "/";
-    } else if (window.APP.baseUrl === "/autobrr/") {
-      baseUrl = "/autobrr/";
+    if (window.APP.baseUrl === "{{.BaseUrl}}") {
+      baseUrl = ""; // Use an empty string for local development
```

Comment:
This might need to be `/`. Still need to further test but without it my local dev env seem to break on hard refresh on nested routes like `/filters/id/actions`.

For built binaries and baseUrl it does not seem to matter much.

Reply:
hmm okay, well let me know if we need it then I will add it.

Reply:
The easiest would be just to leave it as it was. Otherwise, you can fix the baseUrl quirk in APIClient.ts:
```
  let base = baseUrl();
  if (base.slice(-1) !== "/")
    base += "/";

  return window.fetch(`${base}${endpoint}`, config)
```
(which wouldn't be a bad thing by itself either)